### ISSUE TITLE
When Overflow, SubMenu not showing

### ISSUE THREAD

**Issue by husleuujii** [szhsin/react-menu#88]
[https://i.imgur.com/QZXtmme.png](url) this screentshot shows overflowing submenu , I m only able to see overflow x to submenu othervise submenu

**Comment by szhsin** [comment]
Hi @husleuujii , can you please give more details of the issue, does the container have css `overflow` property?

**Comment by gang-qiu-social-chorus** [comment]
hey @szhsin ! love the library so far. But I'm experiencing the same issue. The problem arises when ..
- overflow: auto is set on the menu component
- a long list is rendered (scroll vertical is fine)
- and there is a submenu present (a horizontal scroll appears)

It seems like the top level menu component is accounting for the horizontal scroll of child menus as well. Something like this...

```
        <Menu menuButton={<MenuButton>Open menu</MenuButton>} overflow="auto" position="initial">
            <SubMenu label="Open">
                <MenuItem>index.html</MenuItem>
                <MenuItem>example.js</MenuItem>
                <MenuItem>about.css</MenuItem>
            </SubMenu>
            <MenuItem>Save</MenuItem>
            {new Array(1000).fill(0).map(
                 (_, i) => <MenuItem key={i}>Item {i + 1}</MenuItem>)}
       </Menu>
```

Ideally, each menu/submenu should be absolutely positioned elements with their own scrollable viewports

**Comment by gang-qiu** [comment]
Hey @szhsin , loving the library so far! But I am running into this issue as well and not sure if there's a way to fix it. The problem arises when the top-level menu is overflowing (vertical scroll kicks in, cool) AND it contains a submenu, which will then trigger a horizontal scroll (not so cool). See this sandbox.  https://codesandbox.io/s/vigorous-driscoll-2o7oj?file=/src/App.js

For now I am just going to try to avoid putting submenus on lists that I anticipate will be overflowing. Note that if a submenu overflows, then all is well. However, ideally all menus should be able to independently scroll vertically without the awkward horizontal scroll.

Anyway, keep up the good work.

**Comment by husleuujii** [comment]
@gang-qiu yeah we got same issue, thank you for well explained, I was busy to respond.hehe

**Comment by szhsin** [comment]
@gang-qiu @husleuujii , thanks for the feedback.

This is a limitation of the `overflow` props which has been pointed out in the document: an `overflow` value other than 'visible' can only be set on the last level of menu cascading.

As this feature does offer value in certain scenarios I'll fix it in a coming release.

**Comment by MFQ** [comment]
@szhsin any solution to this problem ?

**Comment by szhsin** [comment]
Hi @MFQ , I'm currently working on v2.0 with some major feature updates and code refactoring. Hopefully this issue will be fixed in the next major version release.

**Comment by szhsin** [comment]
Issue fixed in v2.0.0
